src/css/styles/_c-banner.scss

```diff
@@ -0,0 +1,82 @@
+.c-banner {
+  position: relative;
+  width: 100%;
+  min-height: 568px;
+  max-height: 1440px;
+  margin: 0 auto;
+  margin-bottom: 6rem;
+  background: url("../img/c-banner__bg.jpg") no-repeat 40% 60%;
+  background-size: auto 130%;
+
+  &__title {
+    margin-bottom: 3rem;
+    font-size: 4rem;
+    line-height: 120%;
+    text-shadow: 0 0 10px $col-bg;
+  }
+
+  &__txt {
```

Comment:
- [ ] `c-banner__txt` vs `c-banner__description` (name)

Reply:
Well, I don't mind- if you think it's better, let's go do it this way ;)